packages/edit-site/src/components/header-edit-mode/more-menu/index.js

```diff
@@ -120,7 +120,7 @@ export default function MoreMenu( { showIconLabels } ) {
 						</MenuGroup>
 						<ModeSwitcher />
 						<ActionItem.Slot
-							name="core/edit-site/plugin-more-menu"
```

Comment:
Would it make sense to have two slots here one with core/editor/plugin-more-menu and another one with the old one core/edit-site/plugin-more-menu in case a plugin is using the old "core/edit-site/plugin-more-menu" directly?

Reply:
I'd rather not personally, I'm for unifying the editors entirely including their extensibility APIs as one editor will ultimately replace the other.